alpha/hal.cpp

```diff
@@ -331,38 +330,41 @@ void _TS_HAL::log_init()
 //
 // WIFI setup
 // 
-void _TS_HAL::wifi_init()
+void _TS_HAL::wifi_connect()
```

Comment:
why does code here look copied to radio.cpp

Reply:
I shifted it over to radio.cpp because we wanted to seperate it ? Not sure if I misunderstood